### ISSUE TITLE
Find enemy troops on battlefield

### ISSUE THREAD

**Issue by astranberg** [astranberg/CRBot#1]
We need to be able to find troops on the battlefield, so we can counter them with our own.

**Comment by techdude18** [comment]
Yeah, but are we seriously going to list down all the possible counters per troop, because that's gonna take awhile...
